docs/content/devguide/097-rest.adoc

```diff
@@ -0,0 +1,63 @@
+[[rest]]
+<<<
+== REST Server
+
+=== REST Interface
+
+GeoWave exposes an REST interface to enable some of its commands to be called over HTTP. The total list of operations that can be accessed using this interface is available at the server homepage at `http://localhost:5152`. To start the server, run the following command from within the `geowave/services/rest` directory:
+
+[source, bash]
+----
+mvn install
+mvn exec:java -Dexec.mainClass="mil.nga.giat.geowave.service.rest.RestServer"
+----
+
+=== Authentication
+The REST server provides a simple HTTP authentication mechanism to control access to the service. Upon starting for the first time, the server will create a passwords.csv file in the geowave config directory (i.e. `.geowave/passwords.csv`), with default credentials `admin` and `password`:
+
+[source, bash]
+----
+admin,5e884898da28047151d0e56f8dc6292773603d0d6aabbdd62a11ef721d1542d8 #SHA-256 of "password"
+----
+
+To add another user, modify this file and add the user and password hash of the new user (a future command may be implemented to do this more easily), and restart the server.
+
+=== Support for New Commands
+To enable access to the REST server for a new command, perform the following steps:
+
+. Ensure that the `geowave-service-rest` project has a dependency on the new command. All classes which the REST server can access are visible from the homepage at `http://localhost:5152`, and should appear in [blue]#blue# if they are not yet implemented. 
+. Ensure that the class implementing the operation extends `DefaultOperation`. This is necessary because `DefaultOperation` extends `ServerResource`, which restlet requires for all endpoints.
```

Comment:
'which restlet requires' -> this is the first mention of restlet, which people might be unfamiliar with, I think just 'which the REST server requires' could be more clear